test/unit/api_spec.js

```diff
@@ -3923,6 +3923,20 @@ Caron Broadcasting, Inc., an Ohio corporation (“Lessee”).`)
       expect(items[1].fontName).not.toEqual(items[0].fontName);
     });
 
+    it("get the text a content stream containing some ActualText", async function () {
+      const loadingTask = getDocument(buildGetDocumentParams("issue20007.pdf"));
```

Comment:
I don't really know why, but the unit test failure suggests that this can't be loaded:

`TEST-UNEXPECTED-FAIL | get the text a content stream containing some ActualText | in firefox | ResponseException: Unexpected server response (404) while retrieving PDF "http://127.0.0.1:38175/test/pdfs/issue20007.pdf". in http://127.0.0.1:38175/src/shared/util.js (line 501) `

Moreover, is the movement in the reference test expected?